lightblue/src/test/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepositoryTest.java

```diff
@@ -238,8 +239,9 @@ public void shouldRetrieveNonOverlappingSetsOfNotificationsIfCalledByMultipleThr
             bothThreadsStarted.await();
 
             Thread.sleep(5000);
-
             thread2Client.unpause();
+            
+            Thread.sleep(5000);
```

Comment:
Thanks for this tip. All tests ran flawlessly after adding this...:) 
Removed sleep before unpausing both clients.